FlashEnglish/Sources/Views/Score/ScoreView.swift

```diff
@@ -8,12 +8,17 @@
 import SwiftUI
 
 struct ScoreView: View {
+
+
+
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/f0c1bb17c92f6625d748b7be15707d1134141005/FlashEnglish/Sources/Views/Score/ScoreView.swift#L15"></span>Limit vertical whitespace to a single empty line; currently 5<br />
<code>vertical_whitespace</code> <code>ScoreView.swift:15</code></td>
    </tr>
  </tbody>
</table>